node-js-web-server/index010.js

```diff
@@ -0,0 +1,14 @@
+const http = require('http');
+
+// In order to create your first server, we need a http package.
```

Comment:
This could be broken down more:

* Lesson 1:
"After including a module, you can use any of the methods attached to it. For example, `http.Agent();` uses the `Agent` method. Use the `http` module's `createServer` method to build a server."

* Lesson 2:
"The server is running, but won't do anything if it gets a request. To deal with any requests to the server, first pass a callback function to `createServer`. Here's an example of passing a callback function to `setTimeout`:
```
setTimeout(function() {

});
```

* Lesson 3:
"The callback function in `createServer` takes two parameters, `request` and `response`. Pass the two parameters to the callback function like so:
```
http.createServer(function(req, res) {

});
```

* Lesson 4, 5, and beyond:
I don't think it's necessary to have the student log req.url and req.method to the console. Instead, they could write something like this over a few lessons: 
```
if (req.url === '/') {
  res.write('Hello World');
  res.end();
}
```

Then we can create a simple page to show the "Hello World" message so that it's available offline. That, or students could just write `res.end("Hello World")` and see the simple page.